scripts/standardise_validate.py

```diff
@@ -24,6 +25,15 @@ def str_to_bool(value: str) -> bool:
     raise argparse.ArgumentTypeError(f"Invalid boolean (must be exactly 'true' or 'false'): {value}")
 
 
+def str_to_list_or_none(values: str) -> list[Decimal] | None:
+    if values is None or values == "":
```

Comment:
If you don't expect a `0`, you could simplify?

```suggestion
    if not values:
```

Reply:
This is user input, ... I don't _expect_ `0` but eventually I will receive `0` 🤣

Reply:
With this change, the impact will be that if someone specifies 0 as `scale_to_resolution` the script will not raise an exception - which is incorrect, but probably acceptable.